src/convertToObject.js

```diff
@@ -3,16 +3,32 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+  const splitedString = sourceString.replace(/\n/g, '').split(';');
```

Comment:
```suggestion
  const splitedString = sourceString
    .replace(/\n/g, '')
    .split(';');
```
![image](https://user-images.githubusercontent.com/96409563/233277154-5d4a895d-0180-4ddb-85af-9b1b804e7d37.png)